align/pnr/toplevel.py

```diff
@@ -426,7 +426,7 @@ def gen_leaf_bbox_and_hovertext( ctn, p):
                 (pathlib.Path(opath) / f'{concrete_name}.placement_verilog.json').write_text(scaled_placement_verilog_d.json(indent=2,sort_keys=True))
 
                 standalone_overlap_checker( scaled_placement_verilog_d, concrete_name)
-                check_placement( scaled_placement_verilog_d)
+                check_placement( scaled_placement_verilog_d, scale_factor)
```

Comment:
I would like that scale factor should be 1 here, because "scaled_placement_verilog_d" has already been converted to layers.json units.

Reply:
That is correct. I am using scale_factor to convert to nm as constraints.py also converts um to nm.